.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+
+- Provide a summary of the modification as well as the issue that has been resolved. List any dependencies that this modification necessitates.
+
+- Fixes # (issue)
+
+## Changes
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Checklist:
+
+- [ ] My code follows the pre-commit style and check guidelines of this project.
```

Comment:
Maybe we could add a link to the pre-commit guide that we have in our documentation